### ISSUE TITLE
Import list from domaincostclub.com??

### ISSUE THREAD

**Issue by bendtherules** [judge2020/Actual-Domain-Prices#3]
domaincostclub.com is another at-cost registrars, but have paid membership. List is available openly though.
Maybe consider getting their list also and comparing costs or showing the lowest of them?? They support more tlds like .app, .in from what I checked.

**Comment by rizdaprasetya** [comment]
@bendtherules 
Good idea, because it remained as open issue on this repo, I made my own list to implement that:
https://github.com/rizdaprasetya/Raw-Domain-Extension-Price
Also I used GoogleSheet instead to improve interface & accessibility.

Thanks to the `@judge2020` as the original author of this repo as well. Great project! 🎉
